docs/designers-developers/developers/tutorials/block-based-themes/README.md

```diff
@@ -0,0 +1,375 @@
+	|__ sidebar.html
+	|__ ...
+```
+
+## Creating the theme
+
+Create a new folder for your theme in /wp-content/themes/.
+Inside this folder, create the block-templates and block-template-parts folders.
+
+Create a style.css file.
+The file header in the style.css file has the same items that you would use in a traditional theme.
+https://developer.wordpress.org/themes/basics/main-stylesheet-style-css/#explanations
+
+```
+/*
+Theme Name: My first theme
+Theme URI: 
+Author: The WordPress team
+Author URI: https://wordpress.org/
+Description: 
+Tags:
+Version: 1.0.0
+Requires at least: 5.0
+Tested up to: 5.4
+Requires PHP: 7.0
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Text Domain: myfirsttheme
+
+This theme, like WordPress, is licensed under the GPL.
+Use it to make something cool, have fun, and share what you've learned with others.
+*/
+```
+
+Create a functions.php file.
+
+In this file, you will enqueue the style.css file and add any theme support that you want to use.
+For example colors, wide blocks and featured images.
+
+-You no longer need to add theme support for the title tag. It is already enabled with full site editing.
+
+https://developer.wordpress.org/themes/basics/theme-functions/#what-is-functions-php
+
+https://developer.wordpress.org/block-editor/developers/themes/theme-support/
+
+```php
+<?php
+if ( ! function_exists( 'myfirsttheme_setup' ) ) :
+/**
+ * Sets up theme defaults and registers support for various WordPress features.
+ *
+ * Note that this function is hooked into the after_setup_theme hook, which runs
+ * before the init hook. The init hook is too late for some features, such as indicating
+ * support post thumbnails.
+ */
+function myfirsttheme_setup() {
+ 
+	/**
+	 * Add default posts and comments RSS feed links to <head>.
+	 */
+	add_theme_support( 'automatic-feed-links' );
+ 
+	/**
+	 * Enable support for post thumbnails and featured images.
+	 */
+	add_theme_support( 'post-thumbnails' );
+
+	add_theme_support( 'editor-color-palette', array(
+		array(
+			'name' => __( 'strong magenta', 'myfirsttheme' ),
+			'slug' => 'strong-magenta',
+			'color' => '#a156b4',
+		),
+		array(
+			'name' => __( 'very dark gray', 'myfirsttheme' ),
+			'slug' => 'very-dark-gray',
+			'color' => '#444',
+		),
+	) );
+
+	add_theme_support( 'wp-block-styles' );
+
+	add_theme_support( 'align-wide' );
+}
+endif; // myfirsttheme_setup
+add_action( 'after_setup_theme', 'myfirsttheme_setup' );
+
+function myfirsttheme_scripts() {
+	wp_enqueue_style( 'myfirsttheme-style', get_stylesheet_uri() );
+
+	if ( is_singular() && comments_open() && get_option( 'thread_comments' ) ) {
+		wp_enqueue_script( 'comment-reply' );
+	}
+}
+add_action( 'wp_enqueue_scripts', 'myfirsttheme_scripts' );
+```
+
+Create an index.php file.
+This file is used as a fallback if the theme is activated when full site editing is not enabled. 
+You may leave the file empty for this tutorial.
+
```

Comment:
Before moving on to the next section, would it be helpful to include a quick overview of what the file structure should look like at this point? 

```
 theme
 |__ style.css
 |__ functions.php
 |__ index.php
 |__ block-templates
 	|__ (empty folder)
 |__ block-template-parts
 	|__ (empty folder)
 ```